src/download-overrides.py

```diff
@@ -41,13 +42,24 @@ def assert_epochs_match(overrides_epoch: int, rpmfile_epoch: str):
                         f" and overrides file entry ({overrides_epoch})")
 
 
-assert os.path.isdir("builds"), "Missing builds/ dir; is this a cosa workdir?"
+# You can optionally pass a parameter to specify the overrides directory.
+# If not provided, the default is 'overrides/rpm'
+overrides_dir = sys.argv[1] if len(sys.argv) > 1 else "overrides/rpm"
+print(f"Download override rpms to {overrides_dir}/")
+
+if not os.path.isdir("builds"):
+    print("Warning: Missing 'builds/' directory required by 'cosa build'. "
+          "Safe to ignore if you're not running 'cosa build'.")
 
 rpms = set()
-os.makedirs('overrides/rpm', exist_ok=True)
-for filename in os.listdir(os.path.join("src/config")):
+os.makedirs(overrides_dir, exist_ok=True)
+if os.path.isdir("src/config"):
+    config_dir = "src/config"
+else:
+    config_dir = "."
```

Comment:
hmm. I feel like we are encoding too much in the script here and maybe it would be better if we added options.

```
--destdir ./myoverrides
--lockfiledir ./dir
```

Reply:
and these would have default values of `overrides/rpm` and `src/config`.